package.json

```diff
@@ -35,5 +35,13 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "eslint": "^7.32.0",
+    "eslint-config-standard": "^16.0.3",
+    "eslint-plugin-import": "^2.25.4",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-promise": "^5.2.0",
+    "eslint-plugin-react": "^7.29.3"
```

Comment:
Do we need these packages? You seem to only be using the react plugin.

Reply:
@ANDREYDEN actually i followed a procedure and the ran commands for setting up eslint ...that installed this packages ...🤔i don't have much idea as if all are necessary

Reply:
@RishiKumarRay which procedure did you follow?

Reply:
@ANDREYDEN check out this https://eslint.org/docs/user-guide/getting-started , i was able to setup eslint by going trough this

Reply:
@RishiKumarRay, yes, that's a good guide to follow, but I still don't get where all of those plugin packages came from. The only thing you need is `eslint`.

Reply:
@ANDREYDEN let me keep that plugin only